wagtail/admin/templates/wagtailadmin/home/locked_pages.html

```diff
@@ -31,7 +31,11 @@
                             </div>
                             <ul class="actions">
                                 {% if can_remove_locks %}
-                                    <li><button data-action-lock-unlock data-url="{% url 'wagtailadmin_pages:unlock' page.id %}" class="button button-small button-secondary">{% trans "Unlock" %}</button></li>
+                                    <li><button class="button button-small button-secondary"
+                                        data-action="w-action#post"
+                                        data-controller="w-action"
+                                        data-w-action-redirect-value="false"
```

Comment:
Be careful using false here, you have set your value type as a string on the controller, so this will be parsed as the string false not the Boolean false. 

I recommend you use Stimulus default value approach for the redirect and default it to an empty string. Then just do not set it when it's not meant to be used.

https://stimulus.hotwired.dev/reference/values#default-values

Example. 

```
static values = {
    redirect: { type: String, default: '' },
```

This way you can base your logic on the string being falsey (empty).

Reply:
How does the default value work exactly?
Its not explained well in the stimulus doc.
Setting the default to an empty string here, when setting `data-w-action-redirect-value` would it be `= ""` ?

Reply:
Have a good read of the Stimulus docs for how this works. 

But, the idea of the default value is you simply omit the data attribute when you don't need it. Then the default will be used. 

This is the same approach (but for a Boolean) that was used for the ltsOnly value in the upgrade notification controller.